examples/read-only-auth/docker-compose.yml

```diff
@@ -3,8 +3,8 @@ services:
   registry:
     image: registry:2.7
     volumes:
-      - ./registry-data:/var/lib/registry
-      - ./registry.yml:/etc/docker/registry/config.yml
+      - ./data/registry:/var/lib/registry:rw
+      - ./conf/registry/config.yml:/etc/docker/registry/config.yml:ro
```

Comment:
The real filename is still `registry.yml` and not `config.yml`

Reply:
Didn't understand you If you think "/" is mandatory to say docker it is folder mounting, then no, this is folder mounting, docker will create folder if it is not a file present with same name :)

Reply:
Ah ouch sorry, registry.yml didnt dnoticed that